tests/phpunit/CRM/Contribute/BAO/ContributionRecurTest.php

```diff
@@ -485,6 +473,7 @@ public function validateAllCounts(int $membershipId, int $count): void {
     $lineItemParams = [
       'entity_id' => $membershipId,
       'entity_table' => 'civicrm_membership',
+      'contribution_id' => ['>' => 0],
```

Comment:
Because deleting a contribution leaves orphan line items - which might or might not be a good thing but out of scope of this change